package/objects/TV_Serie__c.object

```diff
@@ -0,0 +1,367 @@
+                    <label>Romance</label>
+                </value>
+                <value>
+                    <fullName>Biography</fullName>
+                    <default>false</default>
+                    <label>Biography</label>
+                </value>
+                <value>
+                    <fullName>Musical</fullName>
+                    <default>false</default>
+                    <label>Musical</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+        <visibleLines>4</visibleLines>
+    </fields>
+    <fields>
+        <fullName>Logo__c</fullName>
+        <externalId>false</externalId>
+        <formula>IMAGE(Logo_url__c , &quot;logo&quot;, 100,100)</formula>
+        <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+        <label>Logo</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Logo_url__c</fullName>
+        <externalId>false</externalId>
+        <label>Logo_url</label>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Url</type>
+    </fields>
+    <fields>
+        <fullName>Rating__c</fullName>
+        <externalId>false</externalId>
+        <formula>Rating_sum__c / Reviews__c</formula>
+        <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+        <label>Rating</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Rating_sum__c</fullName>
+        <externalId>false</externalId>
+        <label>Rating_sum</label>
+        <summarizedField>Review__c.Rating__c</summarizedField>
+        <summaryForeignKey>Review__c.TV_Serie__c</summaryForeignKey>
+        <summaryOperation>sum</summaryOperation>
+        <trackTrending>false</trackTrending>
+        <type>Summary</type>
+    </fields>
+    <fields>
+        <fullName>Reviews__c</fullName>
+        <externalId>false</externalId>
+        <label>Reviews</label>
+        <summaryForeignKey>Review__c.TV_Serie__c</summaryForeignKey>
+        <summaryOperation>count</summaryOperation>
+        <trackTrending>false</trackTrending>
+        <type>Summary</type>
+    </fields>
+    <fields>
+        <fullName>Trailer__c</fullName>
+        <externalId>false</externalId>
+        <label>Trailer</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Url</type>
+    </fields>
+    <fields>
+        <fullName>Year_of_production__c</fullName>
+        <externalId>false</externalId>
+        <label>Year_of_production</label>
+        <length>4</length>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
```

Comment:
Maybe change it into number without decimal places

Reply:
Text type display a number without a comma ;)